dev-libs/processor-trace/processor-trace-1.6.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Intel(R) Processor Trace decoder library"
+HOMEPAGE="https://github.com/01org/processor-trace"
+SRC_URI="https://github.com/01org/processor-trace/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT=0
+KEYWORDS="-* ~amd64"
+IUSE="doc"
+
+DEPEND="doc? ( app-text/pandoc )"
+
+src_configure() {
+	local mycmakeargs=(
```

Comment:
You seem to miss some potentially useful options. From a quick look:

```
option(PTDUMP "Enable ptdump, a packet dumper")
option(PTXED  "Enable ptxed, an instruction flow dumper")
option(PTTC   "Enable pttc, a test compiler")
option(PTUNIT "Enable ptunit, a unit test system and libipt unit tests")
```

FWICS, you need to enable them to run tests. Is there any reason not to run tests in this package (do they work without root privs)? If not, then please add support for them.

Reply:
Fixed. Test works correctly with `PTUNIT=on` only.

p.s. Other options does not intended to be installed, because there's no related stuff in cmake files for them.